nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -77,6 +77,16 @@ let
     else
       config.boot.loader.timeout * 10;
 
+  # Timeout in grub is in seconds.
+  # null means max timeout (infinity)
+  # 0 means disable timeout
+  grubEfiTimeout = if config.boot.loader.timeout == null then
+      -1
+    else
+      config.boot.loader.timeout;
+
+  max = x: y: if x > y then x else y;
```

Comment:
```suggestion
```

Reply:
Gah, sorry I had removed that but goofed on the rebase. Perhaps shouldn't rebase after night shifts :) Fixed again.